maliput/src/api/mod.rs

```diff
@@ -641,6 +641,13 @@ impl<'a> Lane<'a> {
         }
     }
     /// Get the inertial position of the `Lane` at the given `LanePosition`.
+    ///
+    /// Note there is no constraint for the `r` coordinate, as it can be outside the lane boundaries.
+    /// In that scenario, the resultant inertial position represents a point in the `s-r` plane at the given `s` and `h`
+    /// coordinates. It's on the user side to verify, if needed, that the lane position is within lane boundaries.
+    /// Bare in mind that the inertial position will be a point in the `s-r` plane, but *not* necessarily on the road surface.
+    ///
+    /// The s component of lane_position must be in domain [0, Lane::length()].
```

Comment:
Please add the typical args / return comments for methods

Reply:
Later on, we should do that for all the methods and structs in the API, so as not force the user to go to the Cpp docs.
https://github.com/maliput/maliput-rs/issues/179

Reply:
Gotcha. I followed the rest of the API here, but +1 to work on improving this.

Reply:
Yeah I know the rest is missing on this, but we should enforce it, so let's start with this method at least